client/store.js

```diff
@@ -134,10 +147,20 @@ const category = (state = { categories: [] }, action) => {
   }
 };
 
+const review = (state = { reviews: [] }, action) => {
```

Comment:
The reducers were not originally set up with the correct syntax.  I have the correct one on my branch.

When we have separate reducers that we combine, the name of the reducer (e.g. `review`) becomes the property key in state.  So in this case it would be `state.review`, so no need to add a key in the initial state.

```suggestion
const reviews = (state = [], action) => {
```